src/shell/store/notifications.ts

```diff
@@ -8,15 +14,21 @@ export function notifications(state = [], action) {
         if (notification.epoch === action.epoch) {
           notification.active = false;
         }
-        return notificaiton;
+        return notification;
       });
 
     default:
       return state;
   }
 }
 
-export function notify(data) {
+type Data = {
```

Comment:
I think Data is too arbitrary and not indicative enough of what this is. Shouldn't we should name this Notification? Especially in the case we ever need to export this type to a component